Scripts/Python/hopsan.py

```diff
@@ -0,0 +1,52 @@
+class hopsan:
+    def __init__(self):
+        import ctypes
+        import os
+        self.hdll = ctypes.cdll.LoadLibrary(os.getenv('HOPSANHOME')+"/bin/libhopsanc.so")
```

Comment:
I would prefer HOPSAN_HOME, that would match many of the other env variables that are used in the build system

Reply:
I'm not sure really. I was inspired by `OPENMODELICAHOME`, which is used by the OpenModelica project. `HOPSANHOME` has also been used previously to find Hopsan from the OMSimulator tool

Reply:
I still think Hopsan should be internally consistent with naming, it is not in many cases, but adding yet an other one feels unnecessary. Or make a function getHopsanHome that checks both cases, using the first valid one ?

Reply:
No, we should only allow one variable to avoid confusion. I will keep `HOPSANHOME` for now since it was already merged in the Julia PR. If we want to change it later we can make a separate PR for all three wrappers (Julia, Python and Matlab).

Reply:
Ok